form_app/lib/src/validation.dart

```diff
@@ -90,7 +90,7 @@ class _FormValidationDemoState extends State<FormValidationDemo> {
                 // A text field that validates that the text is a noun.
                 TextFormField(
                   validator: (value) {
-                    if (value.isEmpty) {
+                    if (value!.isEmpty) {
```

Comment:
How does this validator method handle the null case?

Would `value == null || value.isEmpty` be a better conditional here?

Reply:
How about `value?.isEmpty ?? false`?